api/src/routes.php

```diff
@@ -46,4 +46,10 @@
         ->add(new AuthenticationMiddleware($container, \JMP\Utils\PermissionLevel::ADMIN))
         ->add($jwtMiddleware);
 
-});
\ No newline at end of file
+    $this->get('/users/{id:[0-9]+}', UsersController::class . ':getUser')
+        ->add(new ValidationErrorResponseBuilder())
+        ->add(new Validation($container['validation']['getUser']))
```

Comment:
Remove this validation as the validation is already done by the expression matching of the route:
`/users/{id:[0-9]+`